docs/docs/tutorials/mcp.md

```diff
@@ -62,14 +65,34 @@ McpTransport transport = new HttpMcpTransport.Builder()
     .build();
 ```
 
+For the Docker stdio transport, you first need to add a module to your pom.xml:
+
+```xml
+<dependency>
+    <groupId>dev.langchain4j</groupId>
+    <artifactId>langchain4j-mcp-docker</artifactId>
+</dependency>
+```
+
+Then you need to create a Docker transport:
+
+```java
+McpTransport transport = transport = new DockerMcpTransport.Builder()
+    .image("mcp/time")
+    .dockerHost("unix:///var/run/docker.sock")
+    .logEvents(true) // if you want to see the traffic in the log
+    .build();
+```
+
 ### MCP Client
 
 To create an MCP client from the transport:
 
 ```java
-McpClient mcpClient = new DefaultMcpClient.Builder()
-    .key("MyMCPClient")
-    .transport(transport)
+McpTransport transport = new DockerMcpTransport.Builder()
```

Comment:
This change doesn't make sense to me, here you're supposed to show how to create a client from the transport, but you're showing the transport again